src/components/02_atoms/Widgets/OptionsSelect.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { css } from 'emotion';
+import PropTypes from 'prop-types';
+import NativeSelect from '@material-ui/core/NativeSelect';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+const styles = {
+  select: css`
+    min-width: 182px;
+  `,
+};
+
+const OptionsSelect = ({
+  label,
+  value,
+  onChange,
+  helpText,
+  fieldName,
+  enum: options,
```

Comment:
Mh, this doesn't work yet because I guess we don't pass the schema along, yet

Reply:
Gotcha. I think we should pass along the schema of the field to each widget. This way you could extract the information you need. Does that make sense?

Reply:
Do you want to do this as part of this PR?

Reply:
Sure, I can do that.